mood-analysis.rb

```diff
@@ -19,6 +20,58 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(words)
+  words.gsub(/[!.#,]/, "")
+end
+
+def happy_days(array)
+  entries = 0
+  happy_faces = 0
+  array.each do |line|
+    if happy_faces < 3
```

Comment:
This check is unnecessary because there shouldn't be a situation where happy_faces is greater than of equal to 3 because of the break below.